code/modules/mob/living/carbon/human/life.dm

```diff
@@ -28,25 +28,26 @@
 	if (QDELETED(src))
 		return 0
 
-	if(.) //not dead
-		handle_active_genes()
+	if(!IsInStasis())
+		if(.) //not dead
+			handle_active_genes()
 
-	if(stat != DEAD)
-		//heart attack stuff
-		handle_heart()
+		if(stat != DEAD)
+			//heart attack stuff
+			handle_heart()
 
-	if(stat != DEAD)
-		//Stuff jammed in your limbs hurts
-		handle_embedded_objects()
+		if(stat != DEAD)
+			//Stuff jammed in your limbs hurts
+			handle_embedded_objects()
 
-	if(stat != DEAD)
-		handle_hygiene()
+		if(stat != DEAD)
+			handle_hygiene()
```

Comment:
This can't kill you, and is the last stat check for `human/Life()` other than returning.

Reply:
Since this is the last stat check other than returning it's irrelevant whether it can kill you or not.